pkg/devicemapper/devmapper_wrapper_static_deferred_remove.go

```diff
@@ -0,0 +1,36 @@
+// +build linux,cgo,static_build
```

Comment:
From what I know, device mapper can't really work when it is built statically, as it's not working with udev events this way, so we have disabled devmapper for the static build in commit 84f1c054e66d399.

Reply:
Alright, I'll drop it here too.

Reply:
Alright, I'll drop it here too.